services/awp_azure_onboarding/awp_azure_onboarding.go

```diff
@@ -0,0 +1,209 @@
+package awp_azure_onboarding
+
+import (
+	"fmt"
+	"log"
+	"net/http"
+	"time"
+)
+
+const (
+	awpAzureGetOnboardingDataPath = "workload/agentless/azure/terraform"
+	azureOnboardingResourcePath   = "workload/agentless/azure/accounts"
+	cloudAccountsPath             = "AzureCloudAccount/"
+
+    ScanModeInAccountSub = "inAccountSub"
+    ScanModeInAccountHub = "inAccountHub"
+)
+
+type CreateAWPOnboardingDataRequest struct {
+	CentralizedId              string                    `url:"centralizedId"`
+}
+
+type AgentlessAzureTerraformOnboardingDataResponse struct {
+	Region                                     string `json:"region"`
+	AppClientId                                string `json:"appClientId"`
+	CloudAccountId                          string `json:"CloudAccountId"`
+	CentralizedCloudAccountId               string `json:"CentralizedCloudAccountId"`
+}
+
+type CloudAccountResponse struct {
```

Comment:
as we agreed offline, lets find a generalization solution for all these common structs between azure and aws